concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -83,6 +83,7 @@ resources:
     source:
       uri: https://github.com/alphagov/paas-graphite-statsd-boshrelease.git
       tag_filter: {{cf_graphite_version}}
+      branch: 118272895_bosh20
 
```

Comment:
The new PR (https://github.com/alphagov/paas-graphite-statsd-boshrelease/pull/5) is against `gds_master` branch. Nothing wrong with that, but after that PR is merged, this commit should stay in, specify `gds_master` branch and update commit message.